library/core/src/char/methods.rs

```diff
@@ -45,6 +45,16 @@ impl char {
     #[stable(feature = "assoc_char_consts", since = "1.52.0")]
     pub const UNICODE_VERSION: (u8, u8, u8) = crate::unicode::UNICODE_VERSION;
 
+    /// The maximum number of bytes required to [encode](char::encode_utf8) a
+    /// `char` in UTF-8.
+    #[stable(feature = "max_len", since = "1.63.0")]
```

Comment:
I think these should go in `unstable` with a tracking issue, no?

Reply:
There's basically three possible ways forward:
1. This is an internal code readability change, so the consts are `pub(crate)` and it goes in with a `T-libs` review.
2. This is a new API going in unstable, so it can go in with just a review on a "seems reasonable" basis.
3. This goes in insta-stable, at which point is needs a full `T-libs-api` FCP.

Basically we only do 3 if forced (because some things can't go in unstable).  So please make it unstable with a tracking issue, and then it can just be approved.  (Consider also doing some commit squashing.)